qc-runner/src/init.r

```diff
@@ -27,7 +27,10 @@ load_config <- function(development_aws_server) {
 
     # running in linux needs the IP of the host to work. If it is set as an environment variable (by makefile) honor it instead of the
     # provided parameter
-    development_aws_server = Sys.getenv("HOST_IP", development_aws_server)
+    overriden_server = Sys.getenv("HOST_IP", "")
+    if (overriden_server != "") {
+        development_aws_server = overriden_server
+    } 
```

Comment:
Why is this necessary? Should work fine:

![image](https://user-images.githubusercontent.com/15719520/116769260-9b236c80-a9ef-11eb-94fb-e14a7a4bbea0.png)

Reply:
The thing is that we don't have an unset `HOST_IP`. We have a `HOST_IP` set to an empty string and we ended up with `development_aws_server` set to an empty string.

Reply:
AHHH yes it's getting set in `app.js` even if the makefile doesn't:

``` javascript
....
const value = process.env[varNames[ii]] || '';
const replace = `__${varNames[ii]}__`;
```

tricksy ... good catch!